src/main/kotlin/app/revanced/patches/youtube/shorts/shortsoverlay/ShortsOverlayButtonsPatch.kt

```diff
@@ -23,6 +23,7 @@ object ShortsOverlayButtonsPatch : BaseResourcePatch(
         key = "IconType",
         default = DEFAULT_ICON_KEY,
         values = mapOf(
+            "OG Outline" to "og_outline",
```

Comment:
change this to:

```kotlin
"OriginalOutline" to "originaloutline",
```

Reply:
also, rename the `og_outline` folder to `originaloutline`. otherwise it will not work. The value of the patch option will be converted to lowercase, `OG Outline` is not the lowercase version of `og_outline`.

Right now the default icons will be used.